example_exec/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.13..3.16 FATAL_ERROR)
```

Comment:
Do you know which cmake version introduces `MODULE` and `COMPONENTS` for `find_dependency`? I think this should be our minimum required version. Also, any reason to restrict it to a max?

Reply:
IMHO: Debian stable include only cmake version 3.13. 
This is he only reason for me not to write 3.14!

Reply:
mm alright, I see that Ubuntu 18 ships version 3.10, so maybe we can leave it at 13 or even 10

Reply:
The second version number is not a max supported version, but a max version awareness. It is "we're aware of this version, if you know about it, set policies to `NEW` that <max> knows about". https://cmake.org/cmake/help/latest/command/cmake_policy.html?highlight=cmake_policy#setting-policies-by-cmake-version

Reply:
Alright I see thank you for the clarification :)